features/facebookImport/src/model/importers/direct-key-data-importer.js

```diff
@@ -8,17 +8,16 @@ export default class DirectKeyDataImporter {
     }
 
     async import({ zipFile, facebookAccount }) {
-        facebookAccount[this._dataStorageKey] = this.extractData(
-            await readJSONDataArray(this._dataFileName, this._dataKey, zipFile)
+        const extractedData = await readJSONDataArray(
+            this._dataFileName,
+            this._dataKey,
+            zipFile
         );
-        facebookAccount.addImportedFileName(this._dataFileName);
-    }
 
-    /**
-     * Hook method to allow importers to change the data
-     * that gets places into the Facebook Account
-     */
-    extractData(rawData) {
-        return rawData;
+        facebookAccount[this._dataStorageKey] =
+            "extractData" in this
```

Comment:
FWIW, I found it more clear the way it was before

Reply:
This eliminates a dummy method. It's not exactly the same, though, because it will apply the extractData method if there's one in the super class.